extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1292,80 +880,12 @@ export class Project implements ISqlProject {
 				const fileStat = await fs.stat(file.fsPath);
 
 				if (fileStat.isFile() && file.fsPath.toLowerCase().endsWith(constants.sqlFileExtension)) {
-					await this.addScriptItem(relativePath);
+					await this.addSqlObjectScript(relativePath);
 				} else if (fileStat.isDirectory()) {
-					await this.addFolderItem(relativePath);
-				}
-			}
-		}
-	}
-
-	/**
-	 * Adds all folders in the path to the project and saves the project file, if provided path is under the project folder.
-	 * If path is outside the project folder, then no action is taken.
-	 *
-	 * @param relativeFolderPath Relative folder path to add folders from.
-	 * @returns Project entry for the last folder in the path, if path is under the project folder; otherwise `undefined`.
-	 */
-	private async ensureFolderItems(relativeFolderPath: string): Promise<FileProjectEntry | undefined> {
```

Comment:
just to double check - DacFx handles this and adds a folder if needed, right?

Reply:
[This part](https://msdata.visualstudio.com/SQLToolsAndLibraries/_git/DacFx?path=/Source/Projects/Folders/FolderCollection.cs&version=GBmain&line=34&lineEnd=38&lineStartColumn=1&lineEndColumn=62&lineStyle=plain&_a=contents), right?  After all the individual files are read, it reads the folders that are implicitly defined by files that are included.